app/js/actions/careSetting.js

```diff
@@ -1,51 +1,29 @@
 import axios from 'axios';
 import {
-  INPATIENT_CARESETTING,
-  INPATIENT_CARESETTING_ERROR,
-  OUTPATIENT_CARESETTING,
-  OUTPATIENT_CARESETTING_ERROR,
+  PATIENT_CARESETTING,
+  PATIENT_CARESETTING_ERROR,
 } from './actionTypes';
 import networkError from './networkError';
 
 const contextPath = window.location.href.split('/')[3];
 const apiBaseUrl = `/${contextPath}/ws/rest/v1`;
 
-export function fetchInpatientCareSetting() {
-  return dispatch => axios.get(`${apiBaseUrl}/caresetting?q=inpatient`)
+const fetchPatientCareSetting = () => dispatch =>
+  axios.get(`${apiBaseUrl}/caresetting`)
     .then((response) => {
       dispatch({
```

Comment:
Looks like there is no loading action logic implemented here, is it intention?
I guess the team also agreed on not allowing the dispatch action direction return its objects

Reply:
Code updated to reflect loading and action creator dispatch.